src/ray/raylet/worker_pool_test.cc

```diff
@@ -46,8 +46,8 @@ int POOL_SIZE_SOFT_LIMIT = 3;
 int WORKER_REGISTER_TIMEOUT_SECONDS = 3;
 JobID JOB_ID = JobID::FromInt(1);
 JobID JOB_ID2 = JobID::FromInt(2);
-std::string BAD_RUNTIME_ENV = "bad runtime env";
-const std::string BAD_RUNTIME_ENV_ERROR_MSG = "bad runtime env";
+static const char BAD_RUNTIME_ENV[] = "bad runtime env";
```

Comment:
Not a merge blocking, but would prefer 
```c++
constexpr std::string_view kBadRuntimeEnv = ...;
constexpr std::string_view kBadRuntimeEnvErrorMsg = ...;
```
two benefits:
- `constexpr` resolves at compile time, and allows compile-time computation if necessary (especially after C++20)
- std::string_view provides safer access than char array